posthog/batch_exports/sql.py

```diff
@@ -318,3 +318,22 @@
     SETTINGS optimize_aggregation_in_order=1
 )
 """
+
+# TODO: is this the best query to use?
+EVENT_COUNT_BY_INTERVAL = """
+SELECT
+    toStartOfInterval(_inserted_at, INTERVAL {interval}) AS interval_start,
+    interval_start + INTERVAL {interval} AS interval_end,
+    COUNT(*) as total_count
+FROM
+    events_batch_export_recent(
```

Comment:
`events_batch_export_recent` should only be used with 5 minute batch exports. Other batch exports should use `events_batch_export_unbounded` or `events_batch_export`.

Reply:
For the 5 minute batch export case though, the query looks fine :+1: